app/controllers/chat_channels_controller.rb

```diff
@@ -62,23 +62,20 @@ def unfollow
   end
 
   def create
-    params.require([:type, :id, :name])
+    params.require([:id, :name])
     guardian.ensure_can_create_chat_channel!
-    raise Discourse::InvalidParameters unless params[:type].downcase == 'category'
     raise Discourse::InvalidParameters.new(:name) if params[:name].length > SiteSetting.max_topic_title_length
 
-    chatable_type = 'Category'
-
     exists = ChatChannel.exists?(
-      chatable_type: chatable_type,
+      chatable_type: 'Category',
       chatable_id: params[:id],
```

Comment:
Not related to the change but I find this param name confusing.. I would think `param[:id]` is the id for the chat channel and not the foreign key of `chatable`.

Reply:
Yeah it is a bit odd perhaps we could change that in another PR